src/analysis/connectivity/overall_scores.sql

```diff
@@ -23,7 +23,7 @@ CREATE TABLE generated.neighborhood_overall_scores (
 INSERT INTO generated.neighborhood_overall_scores (
     category, score_original, score_normalized, human_explanation
 )
-SELECT  'People',
+SELECT  'people',
```

Comment:
Changed to score_id. I'd like to leave the explanation fields in there for now. We may actually want to leave them in there for good as a reference for users who download the scores.